ConsumerApi.Sdk/Client.cs

```diff
@@ -18,7 +18,7 @@ namespace Backbone.ConsumerApi.Sdk;
 
 public class Client
 {
-    public Client(Configuration configuration) : this(new HttpClient { BaseAddress = UriUtils.CreateUriFromParts(configuration.BaseUrl, "api", configuration.ApiVersion) }, configuration)
+    public Client(Configuration configuration) : this(new HttpClient { BaseAddress = UriUtils.CreateUriFromParts(configuration.BaseUrl, "api", configuration.ApiVersion, "/") }, configuration)
```

Comment:
Instead of this "hack", I would rather suggest to update the `CreateUriFromParts` method to make sure the Uri always contains a trailing backslash.